doc/indent-example.c

```diff
@@ -0,0 +1,45 @@
+#include "r_core.h"
+#include "r_util.h"
+#include <stdio.h>
+
+typedef struct r_core_rtr_host_t2 {
+	int proto;
+	char host[512];
+	int port;
+	char file[1024];
+	RSocket *fd;
+} RCoreRtrHost2;
+
+static const char *help_msg_aa[] = {
+	"Usage:", "aa[0*?]", " # see also 'af' and 'afna'",
+	"aa", " ", "alias for 'af@@ sym.*;af@entry0;afva'", //;.afna @@ fcn.*'",
+	"aa*", "", "analyze all flags starting with sym. (af @@ sym.*)",
+	NULL};
+
+static int cmpaddr(const void *_a, const void *_b) {
+	const RAnalFunction *a = _a, *b = _b;
+	return a->addr - b->addr;
+}
+
+int main(int argc, char **argv) {
+	r_anal_esil_set_pc (core->anal->esil, fcn ? fcn->addr : core->offset);
+	switch (*input) {
+	case 'a': // "afta"
+	{
```

Comment:
I've seen so many different styles related to switch/cases in r2 codebase that I think we can adapt to this, since at least it is supported by automatic tools.

Reply:
i think we should refactor this out and not allow such blocks in case statements. just move the code in a separate function or define variables outside will make it more readable. so im not sure if i want to define a syntax rule for those cases

Reply:
I do agree, but still, there's so much code in r2codebase that has these code blocks in the switch-cases... And refacatoring is something the formatter can't do, we should ask for it during reviews.

Reply:
coccinelle can do that

Reply:
Can it extract functions from switch-case blocks?

Anyway, I don't think we do want such a behaviour. That's something that should be reviewed manually by us.